templates/21-loggly-application.conf

```diff
@@ -0,0 +1,66 @@
+# {{ ansible_managed }}
+
+$ModLoad imfile
+$InputFilePollInterval 10
+$PrivDropToGroup adm
+
+#Add a tag for nginx events
+$template LogglyFormatNginx,"<%pri%>%protocol-version% %timestamp:::date-rfc3339% %HOSTNAME% %app-name% %procid% %msgid% [{{ loggly.token }}@41058 tag=\"nginx\"] %msg%\n"
+
+$WorkDirectory /var/spool/rsyslog # where to place spool files
+$ActionQueueFileName fwdRule1     # unique name prefix for spool files
+$ActionQueueMaxDiskSpace 1g       # 1gb space limit (use as much as possible)
```

Comment:
if it's 1g per queue could we run into disk space problems? I think we have 4G on our mediums.